package.json

```diff
@@ -37,14 +24,15 @@
     "jest:e2e-watch": "jest -c jest.config.e2e.js --watch --",
     "start": "node src/backend",
     "server": "node src/backend/web/server",
-    "test-ci": "run-s prettier-check test",
-    "pre-commit": "pretty-quick --staged && npm run lint",
+    "test-ci": "pnpm prettier-check && pnpm lint && pnpm test",
```

Comment:
My concern about this change is that I think we had people on Windows who couldn't use `&&` in their shells.  Is there a reason you can't use `run-s`?

Reply:
I'll change it back to `run-s` since we are keeping that dependency.